code/datums/outfits/outfit_admin.dm

```diff
@@ -18,222 +18,436 @@
+	l_pocket = /obj/item/weapon/pinpointer/advpinpointer
+	l_hand = /obj/item/weapon/tank/jetpack/oxygen/harness
 
-/datum/outfit/admin/hardsuit/wizard
-	name = "Wizard Hardsuit"
-	suit = /obj/item/clothing/suit/space/hardsuit/wizard
-	head = /obj/item/clothing/head/helmet/space/hardsuit/wizard
-
-/datum/outfit/admin/hardsuit/medical
-	name = "Medical Hardsuit"
-	suit = /obj/item/clothing/suit/space/hardsuit/medical
-	head = /obj/item/clothing/head/helmet/space/hardsuit/medical
-
-/datum/outfit/admin/hardsuit/atmos
-	name = "Atmos Hardsuit"
-	suit = /obj/item/clothing/suit/space/hardsuit/atmos
-	head = /obj/item/clothing/head/helmet/space/hardsuit/atmos
-
-
-/datum/outfit/admin/tournament
-	suit = /obj/item/clothing/suit/armor/vest
-	shoes = /obj/item/clothing/shoes/black
-	head = /obj/item/clothing/head/helmet/thunderdome
-	r_pocket = /obj/item/weapon/grenade/smokebomb
-	l_hand = /obj/item/weapon/kitchen/knife
-	r_hand = /obj/item/weapon/gun/energy/pulse/destroyer
-
-/datum/outfit/admin/tournament/red
-	name = "Tournament Standard Red"
-	uniform = /obj/item/clothing/under/color/red
+	backpack_contents = list(
+		/obj/item/weapon/storage/box/engineer = 1,
+		/obj/item/weapon/reagent_containers/food/pill/initropidril = 1,
+		/obj/item/weapon/gun/projectile/automatic/pistol = 1,
+		/obj/item/ammo_box/magazine/m10mm = 1,
+		/obj/item/weapon/crowbar/red = 1,
+		/obj/item/weapon/grenade/plastic/c4 = 1,
+		/obj/item/weapon/reagent_containers/food/snacks/syndidonkpocket = 1,
+		/obj/item/device/flashlight = 1,
+	)
 
-/datum/outfit/admin/tournament/green
-	name = "Tournament Standard Green"
-	uniform = /obj/item/clothing/under/color/green
+/datum/outfit/admin/syndicate/operative/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	. = ..()
+	if(visualsOnly)
+		return
 
-/datum/outfit/admin/tournament_gangster //gangster are supposed to fight each other. --rastaf0
-	name = "Tournament Gangster"
+	var/obj/item/weapon/implant/explosive/E = new(H)
+	E.implant(H)
 
-	uniform = /obj/item/clothing/under/det
-	suit = /obj/item/clothing/suit/storage/det_suit
-	shoes = /obj/item/clothing/shoes/black
-	head = /obj/item/clothing/head/det_hat
-	glasses = /obj/item/clothing/glasses/thermal/monocle
-	l_pocket = /obj/item/ammo_box/a357
-	r_hand = /obj/item/weapon/gun/projectile/automatic/proto
+/datum/outfit/admin/syndicate_strike_team
+	name = "Syndicate Strike Team"
 
-/datum/outfit/admin/tournament_chef //Steven Seagal FTW
-	name = "Tournament Chef"
+/datum/outfit/admin/syndicate_strike_team/equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	return H.equip_syndicate_commando()
 
-	uniform = /obj/item/clothing/under/rank/chef
-	suit = /obj/item/clothing/suit/chef
-	shoes = /obj/item/clothing/shoes/black
-	head = /obj/item/clothing/head/chefhat
-	l_pocket = /obj/item/weapon/kitchen/knife
-	r_pocket = /obj/item/weapon/kitchen/knife
-	l_hand = /obj/item/weapon/kitchen/knife
-	r_hand = /obj/item/weapon/kitchen/rollingpin
+/datum/outfit/admin/syndicate/officer
```

Comment:
Not a fan of the changes made to this. https://github.com/ParadiseSS13/Paradise/blob/294a90e9167379af24a47f2f8f01d59ef4c1fb4f/code/modules/admin/verbs/infiltratorteam_syndicate.dm#L117 has the syndi officer loadout, and every item in it is chosen for a reason. The thermal glasses let them see the SIT team through the shutters, opening them at the right time for dramatic effect. The hardsuit is for intimidation and presence - not combat. The 500 TC implant is so that they can spawn a team with 0 TC, and buy items for them to give them specific equipment. The dust implant is a safety measure intended to prevent their gear falling into the wrong hands, should things (e.g: people hijacking the SIT shuttle) happen.